nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Metrics data to ServiceControl
+summary: Reporting metrics to ServiceControl.
+reviewed: 2017-09-26
+component: MetricsServiceControl
+related:
+ - nservicebus/operations
+related:
+ - samples/logging/metrics
+---
+
+
+This component enables sending monitoring data to ServiceControl.Monitoring instance.
```

Comment:
in this context "ServiceControl.Monitoring" is ambiguous and should link to more info

Reply:
> This component enables sending monitoring data to ServiceControl.Monitoring instance.

I'd rephrase it to:

> This component enables sending monitoring data gathered with NServiceBus.Metrics to an instance of ServiceControl.Monitoring service.